src/app/file-browser/components/sidebar/sidebar.component.ts

```diff
@@ -48,7 +48,6 @@ export class SidebarComponent implements OnDestroy, HasSubscriptions {
     private accountService: AccountService,
     private cdr: ChangeDetectorRef,
   ) {
-    console.log(this.selectedItem);
```

Comment:
FWIW I'd just combine this change into the commit that introduced the rogue console log (git rebase and then `f` to fixup)